src/js/tech/flash.js

```diff
@@ -101,7 +95,8 @@ class Flash extends Tech {
     // use stageclick events triggered from inside the SWF instead
     player.on('stageclick', player.reportUserActivity);
 
-    this.el_ = Flash.embed(options['swf'], placeHolder, flashVars, params, attributes);
+    this.el_ = Flash.embed(options['swf'], flashVars, params, attributes);
+    this.el_['tech'] = this;
```

Comment:
We should use dot notation here now that we're not using closure compiler. `this.el_.tech`